doc-content/enterprise-only/openshift/ref-custom-resource-parameters.adoc

```diff
@@ -0,0 +1,70 @@
+
+`name`: dataset name +
+`location`: dataset address, url +
+`password`: dataset password (leave blank if using a token) +
+`user`: dataset username (leave blank if using a token) +
+`replaceQuery`: Enables automatic query creation if you use dashboards on a {KIE_SERVER}. +
+`token`: {KIE_SERVER} token (leave blank if using user and password
+
+For multi datasets, use comma separated values. Dashes are replaced by underscore on environment variable names. The valid values for this prefix are:
+
+${datasetPrefix}_LOCATION, ${datasetPrefix}_USER, ${datasetPrefix}_PASSWORD, ${datasetPrefix}_TOKEN, ${datasetPrefix}_REPLACE_QUERY.
+
+For more information, see https://docs.jboss.org/jbpm/release/7.45.0.Final/jbpm-docs/html_single/#_kie_server_integration. | --
+|`kieServerTemplates` | `KIESERVER_SERVER_TEMPLATES` | Defines the KIE Server Templates access configuration, for example:
+
+`name`: dataset name +
+`location`: dataset address, url +
+`password`: dataset password (leave blank if using a token) +
+`user`: dataset username (leave blank if using a token) +
+`replaceQuery`: Enables automatic query creation if you use dashboards on a {KIE_SERVER}. +
+`token`: {KIE_SERVER} token (leave blank if using user and password
+
+For multi serverTemplate, use comma separated values. Dashes are replaced by underscore on environment variable names. The valid values for this prefix are:
+
+${serverTemplatePrefix}_LOCATION, ${serverTemplatePrefix}_USER, ${serverTemplatePrefix}_PASSWORD, ${serverTemplatePrefix}_TOKEN, ${datasetPrefix}_REPLACE_QUERY.
+
+For more information, see https://docs.jboss.org/jbpm/release/7.45.0.Final/jbpm-docs/html_single/#_kie_server_integration. | --
+|`modelFileRemoval` | `DASHBUILDER_MODEL_FILE_REMOVAL` | Enables automatic removal of model file from the file system. Default value is false. | False
+|`modelUpdate` | `DASHBUILDER_MODEL_UPDATE` | Allows Runtime to check model last update in the file system to update the content. Default value is true. | True
+|`persistentConfigs` | `` | Sets Dashbuilder as not ephemeral. If `ImportFileLocation` is set `PersistentConfigs` is ignored. Default value is true. Available only on operator. | True
+|`runtimeMultipleImport` | `DASHBUILDER_RUNTIME_MULTIPLE_IMPORT` | Allows Runtime to allow imports (multi-tenancy). Default value is false. | False
+|`uploadSize` | `DASHBUILDER_UPLOAD_SIZE` | Sets the size limit for dashboard uploads (in kb). Default value is 10485760 kb. | 10485760
+|`env` | -- | EnvVar represents an environment variable present in a Container. | --
+|===
+
+You can set options using environment variables, for example:
+
+----
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: standalone-dashbuilder
+spec:
+  environment: rhpam-standalone-dashbuilder
+  objects:
+    dashbuilder:
+      env:
+        - name: DASHBUILDER_UPLOAD_SIZE
+        value: '1000'
```

Comment:
indentation not aligned on `value`field.

Reply:
Nice catch